src/components/TooltipNew/PopperTooltip.js

```diff
@@ -340,6 +342,10 @@ class PopperTooltip extends React.PureComponent {
 PopperTooltip.propTypes = {
   children: PropTypes.node.isRequired,
   className: PropTypes.string,
+  /**
+   * The theme changes the look of the tooltip.
+   */
+  theme: PropTypes.oneOf(['normal', 'invert', 'important']),
```

Comment:
IMHO we don't need `normal` theme. `popper-tooltip--normal` is redundant

Reply:
ok